activerecord/lib/active_record/enum.rb

```diff
@@ -35,6 +35,12 @@ module ActiveRecord
   #   Conversation.archived
   #   Conversation.not_archived
   #
+  # Defining scopes can be disabled by setting +:_scopes+ to +false+.
+  #
+  #   class Conversation < ActiveRecord::Base
+  #     enum status: [ :active, :archived ], _scopes: false
+  #   end
+  #
```

Comment:
Can you please move this to after the next paragraph/examples? I think it makes the reading flow a little better to show that you can query before showing you how to disable. Make sure to squash / keep a single commit. Thanks!

Reply:
@carlosantoniodasilva done